test/framework/src/main/java/org/elasticsearch/test/rest/yaml/Features.java

```diff
@@ -39,6 +39,7 @@
             "catch_unauthorized",
             "embedded_stash_key",
             "headers",
+            "stash_in_key",
             "stash_in_path",
```

Comment:
are embedded_stash_key, stash_in_key and stash_in_path features that only the docs runner uses? In that case we would unify them into one single feature maybe?